pkg/policy/k8s/cilium_cidr_group_test.go

```diff
@@ -266,6 +266,96 @@ func TestHasCIDRGroupRef(t *testing.T) {
 			cidrGroup: "cidr-group-2",
 			expected:  true,
 		},
+		{
+			name: "CIDR in Spec (Egress)",
+			cnp: &types.SlimCNP{
+				CiliumNetworkPolicy: &cilium_v2.CiliumNetworkPolicy{
+					TypeMeta: metav1.TypeMeta{
+						APIVersion: "cilium.io/v2",
+						Kind:       "CiliumNetworkPolicy",
+					},
+					ObjectMeta: metav1.ObjectMeta{
+						Name:      "test-policy",
+						Namespace: "test-namespace",
+					},
+					Spec: &api.Rule{
+						Ingress: []api.IngressRule{
+							{
+								IngressCommonRule: api.IngressCommonRule{
+									FromCIDRSet: api.CIDRRuleSlice{
+										{
+											CIDRGroupRef: "cidr-group-1",
+										},
+									},
+								},
+							},
+						},
+					},
```

Comment:
Here we should have a policy with a reference to the `cidr-group-1` in the `Egress` field, not in the "ingress" one.
That's because the test case is about "CIDR in Spec (Egress)". The ability to reference a CIDRGroup in the Ingress field is already tested in the `CIDR in Spec (Ingress)` above.

Reply:
Updated.